examples/Tutorials/Boilerplate/README.md

```diff
@@ -0,0 +1,16 @@
+# Uber Rides iOS SDK - Boilerplate
+
+This boilerplate code already includes Uber SDK on the `podfile`. All you have to do is:
+
+1. Clone this repo.
+2. Go to this folder - `/examples/Tutorials/Boilerplate`
+3. Run this in your terminal:
+```
+pod install
+```
+
+And change these values in your `Info.plist` with values from your [dashboard](https://developer.uber.com/dashboard)
+1. Replace `[Your Client ID]` with your Client ID (found in the *Settings* tab)
```

Comment:
I think some screenshots here would really help with any confusion about the dashboard